### DISCUSSION TITLE
[Suggestion] Support the Project by Donating

### DISCUSSION THREAD

**Discussion by meirroth** [umami-software/umami#620] [General]
After about two months of trying and giving up (#360), I finally got umami to work. I ended up using Vercel and DO.

This project is really awesome, it's just what I needed!

Please consider adding an option to support you and the project by donating money.  Like a link to PayPal or [Buy Me a Coffee](https://www.buymeacoffee.com/) for example.

Thank you!

**Comment by cbluth** [comment]
^^ yes, please.
i plan on donating after i set up my project, too.
an easy way to donate a custom amount would be nice.

**Comment by adudeongithub** [comment]
I would be happy to donate as well. This is a great project.

**Comment by smartpunter** [comment]
Only crypto, only the true way for donations, without middleman.

**Comment by meirroth** [comment]
@smartpunter Not everyone has crypto.
Though this is up for the creator of this project to decide.

**Comment by meirroth** [comment]
@mikecao

**Comment by Fawesum** [comment]
I would also like to support this with a donation!

**Comment by candideu** [comment]
Any updates on this? I'm highlighting the open source software I've used for my project, and wanted to list any donation links

**Comment by ancashoria** [comment]
Hey, we're a small company and we'd like to support Umami. Please give us an option as this is the only way we can trust the project long term. (Cloud subscription doesn't work for us)
